File: src/main/java/com/formonk/workers/models/GetUniqueAuthTokens.java
package com.formonk.workers.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import lombok.Data;
import org.bson.codecs.pojo.annotations.BsonProperty;

@Data
public class GetUniqueAuthTokens {
    @BsonProperty("_id")
    private String _id ; //this is the socialHandleId
    private String accessToken;

    public String getSocialHandleId(){
        return _id;
    }
}

File: src/main/kotlin/com/formonk/workers/models/GetUniqueAuthTokens.kt
package com.formonk.workers.models

import lombok.Data
import org.bson.codecs.pojo.annotations.BsonProperty

@Data
class GetUniqueAuthTokens {
    @BsonProperty("_id")
    val socialHandleId: String? = null //this is the socialHandleId
    var accessToken: String? = null
        set(accessToken) {
            field = this.accessToken
        }
}
